src/common/io/src/borsh_serialization.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Datafuse Labs
```

Comment:
We can rename `io/src/serialization.rs` to `io/src/bincode_serialization.rs`.
We also need a unit test for the borsh_seralization.rs, reference:
https://github.com/datafuselabs/databend/blob/main/src/common/io/tests/it/serialization.rs

BTW, this unit test most parts generated by GPT4.